src/USER-HSMA/HSMA_Example/SPCE_Water.spce-bulk-nvt

```diff
@@ -0,0 +1,44 @@
+# SPC/E water box bulk
```

Comment:
examples for package USER-XXX should be in folder examples/USER/xxx

Reply:
Thanks. Moved these examples to examples/USER/HSMA.